index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>portifolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="header">
+        <div class="style">
+          <div class="right">
+           <h2>9:41</h2>
+          </div>
+          <div class="left">
+           <img src="/images/Cellular Connection.png" alt="">
+           <img src="/images/Wifi.png" alt="">
+           <img src="/images/Battery.png" alt="">
+          </div>
+        </div>
+        <nav class="nav">
+         <h1>My logo</h1>
+        <div class="menu">
+          <img src="/images/Union1.png" alt="">
+        </div>
+        </nav>
```

Comment:
- [ ] Please i would suggest you to wrapp The hamburger button is on the right, and has a `nav` tag.
   * The `nav` tag is for semantics reason, and the right position for convention and user experience for mobile devices.